### DISCUSSION TITLE
Is zi same as oh-my-zsh?

### DISCUSSION THREAD

**Discussion by rahaaatul** [z-shell/zi#313] [Q&A]
I installed OMZ after installing zsh on **Termux**.

And then I installed couple of plugins like [Autosuggestions](https://github.com/zsh-users/zsh-autosuggestions), [Autocomplete](https://github.com/marlonrichert/zsh-autocomplete), [Syntax Highlighting](https://github.com/zsh-users/zsh-syntax-highlighting).

But they made zsh slower. I wanted to make it faster, so I found **zi**. 

But I don't understand how to do it. I did [quick-setup](https://wiki.zshell.dev/docs/getting_started/installation#quick-setup). And then I tried doing:
```
Snippet: OMZ::plugins/zsh-syntax-highlighting
```
But I get this error
```
Downloading: `OMZ::plugins/zsh-syntax-highlighting` (with curl, wget, lftp)…
curl: (56) The requested URL returned error: 404
curl: (56) The requested URL returned error: 404
Error: Download failed…
```
How can I fix this issue? 

I have so many questions but for now I wanna ask is it same as OMZ ? If it is, should I uninstall OMZ? And how do I install and use turbo on all those 3 plugins I linked?

Thank you!

**Comment by Lunarequest** [comment]
hi, so  just someone who uses zi
your syntax is wrong. zsh-syntax-highlighting is not a oh-my-zsh plugin. you want to try something like `zi light zsh-users/zsh-syntax-highlighting`. oh-my-zsh is not related to this project

**Comment by rahaaatul** [comment]
> hi, so just someone who uses zi your syntax is wrong. zsh-syntax-highlighting is not a oh-my-zsh plugin. you want to try something like `zi light zsh-users/zsh-syntax-highlighting`. oh-my-zsh is not related to this project


Hi there! Pardon my inexperience on `Zi`. Isn't `zi light` a `zinit` command (I've seen it in `install.md` of many plugins)?


I was wondering how would I use the turbo feature with those plugins with z-shell. Can you help me with this as I'm almost illiterate in this.
